build/Dockerfile

```diff
@@ -63,6 +63,18 @@ RUN set -x \
 	&& mkdir -p $(dirname $CLUSTER_PATH) \
 	&& chown $STEAM_USER:$STEAM_USER $(dirname $CLUSTER_PATH)
 
+ARG MODS
+ENV INSTALLED_MODS ${MODS}
+
+# Install mods
+RUN set -x \
```

Comment:
I'd prefer moving the `Install mods` block directly below the `Install Don't Starve Together Server` block, if possible.